src/tests.rs

```diff
@@ -53,6 +54,12 @@ fn get_log_name(base: &str) -> String {
     format!("{}-{}-{}", base, current_time.sec, current_time.nsec)
 }
 
+fn delete_log_files(log_name: String) {
+    for log_path in glob::glob(&format!("./{}-*.json", log_name)).unwrap() {
```

Comment:
I feel like this is going to come back to bite us.

Reply:
I'm still trying to figure out the test bug I mentioned above, but I realized that I probably wasn't actually taking down the graph before running `.recover()` in the tests. Now that I do `DeleteOnExit` naturally won't work, since that means the logs are deleted when the first graph is dropped.

I'm currently prefixing the logs with each tests' name, and added a timestamp in https://github.com/mit-pdos/distributary/pull/37/commits/458f854bf3542ae6cb101b95014b7454c7935ff0 to ensure that panics wouldn't break future test runs (since the test files wouldn't get cleaned up). 

Why do you feel like this might cause issues in the future? Definitely open to improvement suggestions!

Reply:
I just get nervous when I see anything that does `rm` and contains a `*`. I'm sure it's fine, especially given that you both prefix and suffix the glob.

Reply:
Where does `delete_log_files` get called from in the new world order? Is it just important for tests (where timestamps in the file names would avoid clashes, even though the developer would have to manually clean the logs), or does it serve a more general purpose?